gtest/src/runner.rs

```diff
@@ -226,7 +226,7 @@ pub fn init_fixture<MQ: MessageQueue, PS: ProgramStorage, WL: WaitList>(
         }
         runner.queue_message(MessageDispatch {
             source_id: message_source,
-            destination_id: message.destination.into(),
+            destination_id: message.destination.clone().into_program_id(),
```

Comment:
It is sub-optimal to clone the whole enum to get the program id only. Note that the `ProgramId` itself is the `Copy` type.

Reply:
Yes, you're right. However `into_program_id` is a consuming method so `clone` of enum is used or we get 'use after move' compile-time error.

Reply:
You are free to add some non-consuming method or implement some std trait for the purpose.

Reply:
ok